.github/workflows/build-image.yml

```
@@ -0,0 +1,41 @@
+name: Build Vigilo image and push to ECR
+on:
+   push:
+      branches:
+        - main
+    # Allow running this workflow manually from the Actions tab
+    workflow_dispatch:
+
+jobs:
+   build_image:
+      name: Build Vigilo image and push to ECR
+      runs-on: ubuntu-latest
+      permissions:
+        id-token: write
+        contents: read
+
+      steps:
+       - name: Checkout
+         uses: actions/checkout@v3
+
+       - name: Configure AWS credentials
+         uses: aws-actions/configure-aws-credentials@v2
+         with:
+            role-to-assume: ${{ env.DEPLOY_IAM_ROLE }}
```

Oh, I thought this was a private repo. Nice catch! Updating it to a secret.

```suggestion
            role-to-assume: ${{ secrets.DEPLOY_IAM_ROLE }}
```